### DISCUSSION TITLE
Запрет повторного скачивания!

### DISCUSSION THREAD

**Discussion by anarchyst47** [lAnubisl/LostFilmTorrentsFeed#23] [General]
Приветствую! Заметил что иногда в RSS-ленту лоста залетают старые серии, возможно они что-то в них меняют (перевод там и пр.), хотя размеры обычно совпадают. И я бы хотел иметь возможность этого избежать, чтобы и повторно ее не качать и даже если она уже на раздаче что-то типо ошибки получается. То есть чтобы сервис, единожды выдав мне конкретную серию больше мне ее не предлагал, надеюсь смог объяснить) 

![osn1](https://user-images.githubusercontent.com/86853763/142186626-a328747c-1bf2-40de-b395-6c651607a351.jpg)
![osn2](https://user-images.githubusercontent.com/86853763/142186647-bf5d61d2-3883-4971-b465-1278cf54dbdf.jpg)

**Comment by anarchyst47** [comment]
Вот на этой картинке будет понятнее, что я имею ввиду - прилетают неактуальные (старые) серии! А поскольку у нас RSS  и автоматизация, они качаются почем зря)

![yellow](https://user-images.githubusercontent.com/86853763/147680881-d2a0073f-eff6-463b-8369-67d7239e7f3a.jpg)

**Reply by lAnubisl** [reply]
Да. я понимаю. Я обязательно с этим что-то сделаю. Проблема в том что они там что-то меняют в этих сериях и они опять попадают в фид

**Reply by sin2044** [reply]
Артисты озвучания приезжают с гастролей или из отпуска и переозвучивают, чтобы персонаж весь сезон говорил уже знакомым зрителям голосом.

**Reply by anarchyst47** [reply]
> Да. я понимаю. Я обязательно с этим что-то сделаю. Проблема в том что они там что-то меняют в этих сериях и они опять попадают в фид

Возможно, бывает и так, но в основном это абсолютно идентичные серии с разными торрент-файлами. Вот как сейчас The Book of Boba Fett. Я раздаю их оба, а файл один)
Всех с новым годом)

![boba](https://user-images.githubusercontent.com/86853763/147838661-d80cd21c-201d-4043-8c38-3072cfc6de06.jpg)
![boba2](https://user-images.githubusercontent.com/86853763/147838759-172b8b49-97a8-47e2-b2e8-61ccd1efe2c2.jpg)

**Reply by lAnubisl** [reply]
Эту проблему я специально пока не решал, но вы сообщите если такое повторится. Я рассмотрю этот случай внимательно и приму меры.

**Reply by lAnubisl** [reply]
Поймал такое поведение.
